assets/src/stories-editor/blocks/amp-story-cta/edit.js

```diff
@@ -22,6 +22,10 @@ import { getBackgroundColorWithOpacity } from '../../../common/helpers';
 import { DraggableText } from '../../components';
 import { STORY_PAGE_INNER_HEIGHT_FOR_CTA } from '../../constants';
 
+// Total padding of top + bottom / left + right.
+const CTA_BUTTON_PADDING_TOP_BOTTOM = 7;
+const CTA_BUTTON_PADDING_LEFT_RIGHT = 38;
```

Comment:
In `amp-stories.css`, `.amp-block-story-cta__link` has `padding: 7px 24px;` - where does `38` come from?

Reply:
This padding is based on padding the values in the editor since the width and height are taken from the editor. The 7px is the top padding that the CTA button has in the editor (there is no bottom padding) and 38 is the left + right padding together (19 + 19).
Additionally, in the editor, there is a wrapper element with extra padding of 5px on the left and right which in total makes 24px as in the FE, however, since the width and height are not taken from the outer wrapper then we only have to deduct 19px.

I'll check if we can take the width and height from the outer wrapper instead to make it more understandable.

Reply:
I think a comment (and maybe variable renaming, maybe vertical and horizontal?) is fine here to explain the origins.

Reply:
Updated within 0eb23d6, let me know if it's more understandable now.

Reply:
Note that the selector is updated as well now, it is actually easier this way, fewer calculations with nested paddings :)

Reply:
Much clearer now indeed, thanks!